src/components/Toggle/Toggle.ts

```diff
@@ -35,7 +40,6 @@ namespace fabric {
     }
 
     private _toggleHandler(): void {
-      this._toggleField.classList.toggle("is-selected");
```

Comment:
`_toggleHandler()` now does nothing yet the method is still used as a callback to a few events.  Was removing this method's body intentional?